qiskit/providers/models/pulsedefaults.py

```diff
@@ -94,44 +101,325 @@ class Command(BaseModel):
+        Return the estimated resonant frequency for the given qubit in Hz.
+
+        Returns:
+            The frequency of the qubit resonance in Hz.
+        """
+        warnings.warn("The qubit frequency estimation was previously in GHz, and now is in Hz.")
+        return self._qubit_freq_est_hz
+
+    @property
+    def meas_freq_est(self) -> List[float]:
+        """
+        Return the estimated measurement stimulus frequency to readout from the given qubit.
+
+        Returns:
+            The measurement stimulus frequency in Hz.
+        """
+        warnings.warn("The measurement frequency estimation was previously in GHz, "
+                      "and now is in Hz.")
+        return self._meas_freq_est_hz
+
+    def ops(self) -> List[str]:
+        """
+        Return all operations which are defined. By default, these are typically the basis gates
+        along with other operations such as measure and reset.
+
+        Returns:
+            The names of all the circuit operations which have Schedule definitions in this.
+        """
+        return list(self._ops_def.keys())
 
-    def build_cmd_def(self):
-        """Construct the `CmdDef` object for the backend.
+    def op_qubits(self, operation: str) -> List[Union[int, Tuple[int]]]:
+        """
+        Return a list of the qubits for which the given operation is defined. Single qubit
+        operations return a flat list, and multiqubit operations return a list of ordered tuples.
+
+        Args:
+            operation: The name of the circuit operation.
+
+        Returns:
+            Qubit indices which have the given operation defined. This is a list of tuples if the
+            operation has an arity greater than 1, or a flat list of ints otherwise.
+        """
+        return [qs[0] if len(qs) == 1 else qs
+                for qs in sorted(self._ops_def[operation].keys())]
+
+    def qubit_ops(self, qubits: Union[int, Iterable[int]]) -> List[str]:
+        """
+        Return a list of the operation names that are defined by the backend for the given qubit
+        or qubits.
+
+        Args:
+            qubits: A qubit index, or a list or tuple of indices.
+
+        Returns:
+            All the operations which are defined on the qubits. For 1 qubit, all the 1Q
+            operations defined. For multiple qubits, all the operations which apply to that
+            whole set of qubits (e.g. qubits=[0, 1] may return ['cx']).
+        """
+        return self._qubit_ops[_to_tuple(qubits)]
+
+    def has(self, operation: str, qubits: Union[int, Iterable[int]]) -> bool:
+        """
+        Is the operation defined for the given qubits?
+
+        Args:
+            operation: The operation for which to look.
+            qubits: The specific qubits for the operation.
+
+        Returns:
+            True iff the operation is defined.
+        """
+        return operation in self._ops_def and \
+            _to_tuple(qubits) in self._ops_def[operation]
```

Comment:
Can we clarify in the doc string what the `qubits` input should look like for single-qubit gates versus multi-qubit gates?

Reply:
Is it not clear from the type hints? There is nothing distinguishing the input for single vs multi qubit gates (except that inherently, it will be difficult to get a multi qubit gate without passing multiple qubits!)